providers/Providers.tsx

```diff
@@ -26,7 +27,9 @@ const Providers = ({ children }: { children: React.ReactNode }) => (
                   <ChatProvider>
                     <PaymentProvider>
                       <FunnelAnalysisProvider>
-                        {children}
+                        <ChatOptionsProvider>
+                          {children}
+                        </ChatOptionsProvider>
```

Comment:
why does the provider need to wrap the entire app if this is only options for the `recent chats` portion of the UI?
- actual: provider wraps full app
- suggested: provider only wraps recent chat component.